CollAction/Views/Projects/Details.cshtml

```diff
@@ -209,7 +212,51 @@
                 </div>
             </div>
         }
+    </div>
+    
+    @if (canTrack)
+    {
+        <div class="container">
+            <div class="row">
+                <div class="col-xs-12">
+                    <hr class="hidden-sm hidden-xs">
+                </div>
+            </div>
+        </div>
 
+        <div id="project-discussions">
+            <div class="container">
+                <div class="row">
+                    <div class="col-xs-12">
+                        <div id="disqus_thread"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        var identifier = $"{@Model.Project.Name}/{@Model.Project.Id}";
+        var script = $"https://{@SiteOptions.Value.DisqusSite}.disqus.com/embed.js";
+        <script>
```

Comment:
I think it is better to put script tags at the bottom of the page, since JS loading blocks the DOM rendering of the rest of the page right?

Reply:
I already moved it to disqus.ts and include it in the general frontend bundle.